src/hooks/useQueryLogs.ts

```diff
@@ -88,24 +86,28 @@ export const useQueryLogs = () => {
 			setError(null);
 
 			const logsQueryRes = isQuerySearchActive
-				? await getQueryResult({ ...logsQuery, access: [] }, appendOffsetToQuery(custSearchQuery, logsQuery.pageOffset))
-				: await getQueryLogs(logsQuery);
+				? await getQueryResult(
+						{ ...logsQuery, access: [] },
+						appendOffsetToQuery(custSearchQuery, logsQuery.pageOffset),
+						{ fields: 'true' },
+				  )
+				: await getQueryLogs(logsQuery, { fields: 'true' });
 
-			const data = logsQueryRes.data;
+			const data = logsQueryRes.data.records;
```

Comment:
There is a possibility where server could just send you {} or null.
Verify you always get a result that contains a records array even if there is no data.